cmd/orb.go

```diff
@@ -23,10 +24,14 @@ var orbAnnotations = map[string]string{
 func newOrbCommand() *cobra.Command {
 
 	listCommand := &cobra.Command{
-		Use:   "list",
-		Short: "List orbs",
-		RunE:  listOrbs,
+		Use:         "list NAMESPACE",
+		Short:       "List orbs",
+		Args:        cobra.MaximumNArgs(1),
+		RunE:        listOrbs,
+		Annotations: make(map[string]string),
 	}
+	optionalOrbs := []string{orbAnnotations["NAMESPACE"], "(Optional)"}
+	listCommand.Annotations["NAMESPACE"] = strings.Join(optionalOrbs, " ")
```

Comment:
Doesn't this break the convention of using flags for optional args and positionals for required args?

If we want to keep this exception the convention for documenting optional positional args is on the `Use` line:

```
list [NAMESPACE]
```

As an FYI, for concating strings in golang this would be more obvious:

```go
orbAnnotations["NAMESPACE"] + " (Optional)"
```

although I would just use the `Use` line.

Reply:
Thanks for the suggestion! @zzak has been doing most of the CLI work and has a strong preference for optional arguments over flags. He called this out as part of this feature's when we were talking about the JIRA ticket, so I'm leaving this as is to maintain consistency.

Reply:
Also thanks for the tip about string concatenation with `+` -- that's much easier to work with!

Reply:
While this does break the convention for optional args, I think using the namespace filter will be common enough and don't see there being a use for another arg in it's position to warrant a flag.

We may want to support something like `orb list certified` which we can re-use the same positional arg in place of namespace to specify a category of orbs to search for.